packages/@uppy/companion/src/server/provider/zoom/adapter.js

```diff
@@ -0,0 +1,50 @@
+const qs = require('qs')
+
+const MIMETYPES = {
+  MP4: 'video/mp4',
+  M4A: 'audio/mp4',
+  CHAT: 'text/plain',
+  TRANSCRIPT: 'text/plain', // note: waiting confirmation, https://devforum.zoom.us/t/retrieve-all-past-zoom-cloud-recordings/22487/5?u=mokutsu
+  CC: 'text/vtt'
+}
+
+exports.getQuery = (results, dateRange) => {
+  const query = {
+    cursor: dateRange.monthsInPast
```

Comment:
does this cursor require a specific format or does it only accept a plain string? It seemed like setting an object to cursor triggered excessive requests for infinite scroll, not sure why